css/styles.css

```diff
@@ -287,8 +466,39 @@ body {
   background-color: #f9f9f9;
 }
 
+@media (min-width: 768px) {
+  .projects-container .project {
+    display: flex;
+    gap: 24px;
+    height: 448px;
+    margin-bottom: 80px;
+  }
```

Comment:
Great design @John-Kibirige 🤝

- [ ] [Optional] You may reduce the height of your cards to make your project a replica of the Figma design provided.